docs/src/aria-snapshots.md

```diff
@@ -329,10 +326,40 @@ When updating snapshots, Playwright creates patch files that capture differences
 applied, and committed to source control, allowing teams to track structural changes over time and ensure updates are
 consistent with application requirements.
 
+The way source code is updated can be changed using the `--update-source-method` flag. There are several options available:
+
+- **"patch"** (default): Generates a unified diff file that can be applied to the source code using `git apply`.
+- **"3way"**: Generates merge conflict markers in your source code, allowing you to choose whether to accept changes.
+- **"overwrite"**: Overwrites the source code with the new snapshot values.
+
+```bash
+npx playwright test --update-snapshots --update-source-mode=3way
+```
+
+#### Snapshot as separate files
+
+To store your snapshots in a separate file, use the `toMatchAriaSnapshot` method with the `name` option, specifying a `.yml` file extension.
+
+```js
+await expect(page.getByRole('main')).toMatchAriaSnapshot({ name: 'main-snapshot.yml' });
+```
+
+By default, snapshots are saved with the following pattern `{snapshotDir}/{testFileDir}/{testFileName}-snapshots/{arg}{ext}`. As snapshots should be the same across browsers, only one snapshot is saved even if testing with multiple browsers. Should you wish, you can customize the snapshot path template using the following configuration:
```

Comment:
I'd link to the API docs that explain what all these `{snapshotDir}` mean. Otherwise it's totally cryptic.